src/README.md

```diff
@@ -221,59 +223,59 @@ class Routine {
-| Function Name | Definition |
------------- | ------------- |
-| getAvailableRoutines | () => Promise\<List\<string\>\> |
+| Function Name | Definition | Released in `dpsl` version |
+------------ | ------------- | ------------- |
+| getAvailableRoutines | () => Promise\<List\<string\>\> | 1.0.0 |
 
 ### dpsl.diagnostics.power.*
-| Function Name | Definition |
------------- | ------------- |
-| runAcPowerRoutine | (params: AcPowerRoutineParams) => Promise\<Routine\> |
+| Function Name | Definition | Released in `dpsl` version |
+------------ | ------------- | ------------- |
+| runAcPowerRoutine | (params: AcPowerRoutineParams) => Promise\<Routine\> | 1.3.1 |
 
 ### dpsl.diagnostics.battery.*
-| Function Name | Definition |
------------- | ------------- |
-| runCapacityRoutine | () => Promise\<Routine\> |
-| runHealthRoutine | () => Promise\<Routine\> |
-| runDischargeRoutine | (params: BatteryDischargeRoutineParams) => Promise\<Routine\> |
-| runChargeRoutine | (params: BatteryChargeRoutineParams) => Promise\<Routine\> |
+| Function Name | Definition | Released in `dpsl` version |
+------------ | ------------- | ------------- |
+| runCapacityRoutine | () => Promise\<Routine\> | 1.0.0 |
+| runHealthRoutine | () => Promise\<Routine\> | 1.0.0 |
+| runDischargeRoutine | (params: BatteryDischargeRoutineParams) => Promise\<Routine\> | 1.0.0 |
+| runChargeRoutine | (params: BatteryChargeRoutineParams) => Promise\<Routine\> | 1.0.0 |
 
 ### dpsl.diagnostics.cpu.*
-| Function Name | Definition |
------------- | ------------- |
-| runCacheRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> |
-| runStressRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> |
-| runFloatingPointAccuracyRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> |
-| runPrimeSearchRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> |
+| Function Name | Definition | Released in `dpsl` version |
+------------ | ------------- | ------------- |
+| runCacheRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> | 1.0.0 |
+| runStressRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> | 1.0.0 |
+| runFloatingPointAccuracyRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> | 1.1.0 |
+| runPrimeSearchRoutine | (params: CpuRoutineDurationParams) => Promise\<Routine\> | 1.1.0 |
 
 ### dpsl.diagnostics.memory.*
-| Function Name | Definition |
------------- | ------------- |
-| runMemoryRoutine | () => Promise\<Routine\> |
+| Function Name | Definition | Released in `dpsl` version |
+------------ | ------------- | ------------- |
+| runMemoryRoutine | () => Promise\<Routine\> | 1.0.0 |
 
 ### dpsl.diagnostics.disk.*
-| Function Name | Definition |
------------- | ------------- |
-| runReadRoutine | () => Promise\<Routine\> |
+| Function Name | Definition | Released in `dpsl` version |
+------------ | ------------- | ------------- |
+| runReadRoutine | () => Promise\<Routine\> | 1.3.0 |
```

Comment:
I would change 1.3.0 -> 1.3.1 since "runReadRoutine" had a bug and cannot be used before 1.3.1

Reply:
Good catch! Thank you :)